docs/reference/builder.md

```diff
@@ -1274,12 +1274,12 @@ The output of the final `pwd` command in this `Dockerfile` would be
     ARG <name>[=<default value>]
 
 The `ARG` instruction defines a variable that users can pass at build-time to
-the builder with the `docker build` command using the
-`--build-arg <varname>=<value>` flag. If a user specifies a build argument
-that was not defined in the Dockerfile, the build outputs an error.
+the builder with the `docker build` command using the `--build-arg
+<varname>=<value>` flag. If a user specifies a build argument that was not
+defined in the Dockerfile, the build outputs a warning.
 
 ```
-One or more build-args were not consumed, failing build.
+[Warning] One or more build-args %v were not consumed.
```

Comment:
Oh, i see you included the literal `%v` in the docs; @duglin meant to say that this should show what it looks like if one or more ARGS were not used